components/brave_wallet/browser/fil_tx_manager.cc

```diff
@@ -224,6 +228,48 @@ FilTxStateManager* FilTxManager::GetFilTxStateManager() {
   return static_cast<FilTxStateManager*>(tx_state_manager_.get());
 }
 
-void FilTxManager::UpdatePendingTransactions() {}
+void FilTxManager::UpdatePendingTransactions() {
+  auto pending_transactions = tx_state_manager_->GetTransactionsByStatus(
+      mojom::TransactionStatus::Submitted, absl::nullopt);
+  for (const auto& pending_transaction : pending_transactions) {
+    auto cid = pending_transaction->tx_hash();
+    uint64_t seconds =
+        (base::Time::Now() - pending_transaction->submitted_time()).InSeconds();
+    uint64_t depth = seconds == 0 ? 1 : seconds;
+    json_rpc_service_->GetFilStateSearchMsgLimited(
+        cid, depth,
+        base::BindOnce(&FilTxManager::OnGetTransactionStatus,
```

Comment:
Should probably just use `OnGetFilStateSearchMsgLimited`.

Reply:
done